test/system/015-help.bats

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bats
+#
+# Tests based on 'podman help'
+#
+# Find all commands listed by 'podman --help'. Run each one, make sure it
+# provides its own --help output. If the usage message ends in '[command]',
+# treat it as a subcommand, and recurse into its own list of sub-subcommands.
+#
+# Any usage message that ends in '[flags]' is interpreted as a command
+# that takes no further arguments; we confirm by running with 'invalid-arg'
+# and confirming that it exits with error status and message.
+#
+load helpers
+
+# run 'podman help', parse the output looking for 'Available Commands';
+# return that list.
+function podman_commands() {
+    dprint "$@"
+    run_podman help "$@" |\
+        awk '/^Available Commands:/{ok=1;next}/^Flags:/{ok=0}ok { print $1 }' |\
+        grep .
+    "$output"
+}
+
+
+function check_help() {
+    count=0
+    for cmd in $(podman_commands "$@"); do
+        dprint "podman $@ $cmd --help"
+        run_podman "$@" $cmd --help
+
+        # FIXME FIXME FIXME
```

Comment:
Now? Later? Intended to remove this line?

Reply:
Need to document.

Reply:
Maybe for now, s/FIXME/TODO/ :smile: